src/main/java/ca/wolfram/beta/symath/operations/MultiplyNode.java

```diff
@@ -45,7 +91,15 @@ public double operationEval(VMap map) {
     public String toString() {
         StringBuilder s = new StringBuilder();
         s.append("(");
-        getChildren().forEach((c) -> s.append(c).append(" * "));
-        return s.substring(0, s.length() - 3) + ")";
+        boolean isFirst = true;
+        for (MathNode c : getChildren()) {
+            if (isFirst) {
+                s.append(c.toString());
+                isFirst = false;
+                continue;
+            }
+            s.append(" * ").append(c.toString());
+        }
```

Comment:
It will get more complex in a bit. I still need to add checkers to see if a child is a power node with ^ -1. In that case it will be /.  Refer to [AddNode().toString()](https://github.com/Wolfram-Beta/SyMath/blob/master/src/main/java/ca/wolfram/beta/symath/operations/AddNode.java#L56) to see the general layout